reference/redirects.v3.yml

```diff
@@ -155,6 +156,215 @@ paths:
+                  description: A list of the redirect IDs you wish to export. If no redirect IDs are provided, the request exports all redirects for the given site selection.
+              description: Data necessary to create a new 301 Redirects export job.
+        required: true
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  id:
+                    type: string
+                    example: ab1c2de3-f4gh-5678-i90j-klm12n345o67
+        '429':
+          description: Your store already has an active Redirects import or export job running.
+        '409':
+          description: No Redirects in your store meet the criteria in your request.
+  '/storefront/redirects/imex/import':
+    parameters:
+      - $ref: '#/components/parameters/Accept'
+    post:
+      tags:
+        - Redirect Import-Export
+      summary: Create Redirects Import Job
+      description: Creates a new 301 Redirects import job.
+      operationId: createRedirectImportJob
+      parameters:
+        - $ref: '#/components/parameters/ContentTypeFormData'
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                import_file:
+                  description: A CSV file containing a list of Redirects to be imported.
+                  type: string
+                  format: binary
+              required:
+                - import_file
+              description: Data necessary to create a new 301 Redirects import job.
+        required: true
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  id:
+                    type: string
+                    example: ab1c2de3-f4gh-5678-i90j-klm12n345o67
+        '400':
+          description: The provided form data was invalid or the file is not a CSV.
+        '413':
+          description: The provided file is too large. The maximum file size is 20MB.
+        '429':
+          description: Your store already has an active Redirects import or export job running.
+  '/storefront/redirects/imex/export/{uuid}/events':
+    parameters:
+      - $ref: '#/components/parameters/AcceptEventStream'
+      - $ref: '#/components/parameters/ImportExportIdParam'
+    get:
+      tags:
+        - Redirect Import-Export
+      summary: Open Redirect Export Event Stream
+      description: Opens an event stream to receive live updates from an export job.
+      operationId: getRedirectExportEvents
+      responses:
+        '200':
+          description: Stream of export events
+          content:
+            text/event-stream:
+              schema:
+                type: string
+                example: |
```

Comment:
we can probably format this example better -- will look into this when i run my requests.